src/sorcha/readers/OrbitAuxReader.py

```diff
@@ -51,21 +52,46 @@ def _process_and_validate_input_table(self, input_table, **kwargs):
         # Do standard CSV file processing
         super()._process_and_validate_input_table(input_table, **kwargs)
 
+        if len(input_table) == 0:
+            return input_table
+
         pplogger = logging.getLogger(__name__)
 
-        # Check for an H column (which should not be in the orbit file).
-        if "H" in input_table.columns:
+        if not "FORMAT" in input_table.columns:
+            pplogger.error("ERROR: PPReadOrbitFile: Orbit format must be provided.")
+            sys.exit("ERROR: PPReadOrbitFile: Orbit format must be provided.")
+
+        if len(input_table.columns) != 9:
+            pplogger.error("ERROR: Please only provide the required columns in your orbits file.")
+            sys.exit("ERROR: Please only provide the required columns in your orbits file.")
+
+        orb_format = input_table["FORMAT"].iloc[0]
+        if not np.all(orb_format == input_table["FORMAT"].values):
```

Comment:
Also minor point - This is conditional is probably fast enough (depending on how many times it's called), but pandas provides a `unique` method that may be more efficient.

`if len(input_table["FORMAT"].unique()) != 1:`

A reference SO response related to using `set`s to find unique values: 
https://stackoverflow.com/a/51227605